src/tools/namespace_tool.cpp

```diff
@@ -94,15 +96,22 @@ int NameSpaceTool::RunCommand(const std::string &cmd) {
             return 0;
         }
     } else if (cmd == kCleanRecycleCmd) {
+        uint64_t expireTime;
+        if (!::curve::common::StringToTime(FLAGS_expireTime, &expireTime)) {
+            std::cout << "Clean recyclebin: invalid expireTime!" << std::endl;
+            return 0;
+        }
+
         if (FLAGS_isTest) {
-            return core_->CleanRecycleBin(fileName);
+            return core_->CleanRecycleBin(fileName.empty() ? "/" : fileName,
+                                          expireTime);
         }
         std::cout << "Are you sure you want to clean the RecycleBin?"
                   << "(yes/no)" << std::endl;
         std::string str;
         std::cin >> str;
         if (str == "yes") {
-            return core_->CleanRecycleBin();;
+            return core_->CleanRecycleBin("/", expireTime);
```

Comment:
return core_->CleanRecycleBin("/", expireTime);
shuold be: return core_->CleanRecycleBin(fileName.empty() ? "/" : fileName, expireTime); ?

Reply:
> return core_->CleanRecycleBin("/", expireTime);
> shuold be: return core_->CleanRecycleBin(fileName.empty() ? "/" : fileName, expireTime); ?

yeap, it's a bug, i will fix it.